.devcontainer/Dockerfile

```diff
@@ -14,3 +14,7 @@ FROM mcr.microsoft.com/vscode/devcontainers/typescript-node:0-${VARIANT}
 
 # [Optional] Uncomment if you want to install more global node packages
 # RUN su node -c "npm install -g <your-package-list -here>"
+
+RUN git clone https://github.com/asdf-vm/asdf.git ~/.asdf --branch v0.8.1 --depth 1
```

Comment:
This is OK for now, but is a common issue across projects to keep in mind 🤔

Reply:
> their repo has a v0.9 branch now

When tracking new version releases of things, the definitive source would be the repo's [releases](https://github.com/asdf-vm/asdf/releases) list. 

So if they have a v0.9 branch now, this would be where they're actively working on the next version. But it's not officially released (ie: ready to use) until that branch is merged to main with a git tag (version number) which will cause it to appear on the list of releases.